unified-codes/tsconfig.base.json

```diff
@@ -17,7 +17,10 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@unified-codes/ui": ["libs/ui/src/index.ts"],
+      "@unified-codes/ui/components": ["libs/ui/src/lib/components/index.ts"],
+      "@unified-codes/ui/styles": ["libs/ui/src/lib/styles/index.ts"],
+      "@unified-codes/ui/types": ["libs/ui/src/lib/types/index.ts"],
+      "@unified-codes/ui/hooks": ["libs/ui/src/lib/hooks/index.ts"],
```

Comment:
Updated the `ui` lib paths to be a bit more specific and to avoid import loops!

Reply:
ah. this is the part required by #226